build.gradle

```
@@ -20,6 +20,9 @@ repositories {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	//Apache Commons Lang・・・便利機能
+	implementation("org.apache.commons:commons-lang3:3.17.0")
```

```suggestion
	implementation 'org.apache.commons:commons-lang3:3.17.0'
```
今のままでもいいですが、書き方を他のライブラリと揃えてもいいかなと思いました。

---

全然気づいていなかったので、修正しました。
ありがとうございます！